pennylane_lightning/src/bindings/Bindings.hpp

```diff
@@ -227,4 +229,57 @@ template <class PrecisionT, class ParamT, class PyClass>
 void registerKernelsToPyexport(PyClass &pyclass) {
     registerKernelsToPyexportIter<PrecisionT, ParamT, 0>(pyclass);
 }
+
+/**
+ * @brief Return basic information of the compiled binary.
+ */
+auto getCompileInfo() -> pybind11::dict {
+    using namespace Util::Constant;
+    using namespace pybind11::literals;
+
+    const std::string_view cpu_arch_str = [] {
+        switch (cpu_arch) {
+        case CPUArch::AMD64:
+            return "AMD64";
```

Comment:
I am also not sure what is the standard. It is also true that this architecture set is devised by AMD. Still, as `x86_64` sounds more neutral, I think changing it may be better (does not make some Intel people angry?).